CONTRIBUTING.md

```diff
@@ -0,0 +1,181 @@
+# Contributing
+
+Contributing to this repository, we follow [Ansible Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html) in all our interactions within the project.
+
+## Issue tracker
+
+Whether you are looking for an opportunity to contribute or you found a bug and already know how to solve it, please go to the [issue tracker](https://github.com/ansible-collections/community.mysql/issues).
+There you can find feature ideas to implement, reports about bugs to solve, or submit an issue to discuss your idea before implementing it which can help choose a right direction initially and potentially save a lot of time and effort.
+Also somebody can already start discussing or working on implementing the same or similar idea,
+so you can cooperate to create a better solution together.
+
+## Open pull requests
+
+Look through currently [open pull requests](https://github.com/ansible-collections/community.mysql/pulls).
+You can help by reviewing them. There can be good pull requests which are not merged only because there are a lack of reviews. Also there can be worth, reviewed, but abandoned ones, which you could politely ask the original authors to complete yourself.
+And it is always worth saying that good reviews are often more valuable than pull requests themselves.
+
+## Looking for an idea to implement
+
+First, see the paragraphs above.
+
+If you came up with a new feature, it is always worth creating an issue
+before starting to write code to discuss the idea with the community first.
+
+## Step-by-step guide how to get into development quickly.
+
+We assume you use Linux as a work environment (you can use a virtual machine as well).
+
+1. Install ``docker``, launch it.
```

Comment:
It's not needed but it's easier to use it rather then struggle with all requirements and dependencies.
It's not different from other collections, I think. It's very convenient to run at least sanity and units with docker.
I think it's fair for all the collections.
The goal, as i mentioned, is to make the first contribution as smooth as possible.

Reply:
How about changing this to something like:
```suggestion
1. If possible, make sure that you have installed and started `docker`. While you can also run tests without docker, this makes it a lot easier since you do not have to install the precise requirements and tests are running properly isolated and in the exact same environments as in CI. You often can also use `podman` with the `docker` executable shim, so if you have that you probably do not need to install `docker`.
```
(MarkDown uses single backticks for code formatting btw :) )

Reply:
Is `podman` mentioning worth? I'm not sure how many users are familiar with `podman` but not `docker`.

I'm saying this for the sake of simplicity and reducing doc length.

Reply:
LGTM.
One minor thing: I'd join @aminvakil 's question about `podman`. I'm not against though.

@felixfontein thanks for pointing out the backticks issue. i saw that the double works as well but didn't know that it's not a standard way. Will change

Reply:
(discussed)